aws-lc-sys/builder/prebuilt-nasm.bat

```diff
@@ -13,7 +13,9 @@ goto loop
 shift
 set "path=%~1"
 for %%f in ("%path%") do set "filename=%%~nxf"
-copy "%ScriptDir%\prebuilt-nasm\%filename%" "%path%"
+REM Extract the base filename before the first dot, similar to the shell script
+for /f "tokens=1 delims=." %%a in ("%filename%") do set "basename=%%a"
+copy "%ScriptDir%\prebuilt-nasm\%basename%.obj" "%path%"
```

Comment:
This change makes the logic consistent what the corresponding [bash script](https://github.com/aws/aws-lc-rs/blob/main/aws-lc-sys/builder/prebuilt-nasm.sh#L12-L13)
```
    filename="$(basename "$path")"
    filename="$(echo "$filename" | cut -f 1 -d '.')"
```